changelog/0037-upgrade-to-elasticsearch7.yml

```diff
@@ -0,0 +1,350 @@
+
+  Use below notes to reindex all of the data up to a day or two before switchover and proceed to the switchover section after that.
+
+   1. The reindex management command should be run from the release that's setup in step 2 of Setup phase above which is pointing to Elasticsearch 7. 
+      - To activate virtual env for reindex commands, SSH into the web worker from Step 2 of Setup phase and do below
+          ```
+          tmux # optional
+          sudo -iu cchq
+          cd <releases_dir_from_setup_phase>
+          source python_env-3.6/bin/activate
+          grep ELASTICSEARCH localsettings.py # to make sure this points to ES7
+          ```
+   2. Reindex all of form/case data up to a planned switchover date
+      - Below is an example management command that reindexes forms index from Elasticsearch 2 cluster at `http://es2_cluster:9200` for a specific date range
+        ```
+          # to get help on usage
+        ./manage.py reindex_es_from_remote --help
+        # an example reindex command
+          ./manage.py reindex_es_from_remote forms http://es2_cluster:9200 --start_date=YYYY-MM-DD
+        --end_date=YYYY-MM-DD 
+        ``` 
+      - Multiple reindex commands with different date ranges can be run in separate tmux/screen session to maximize reindex throughput.
+      - To efficiently split the reindex commands you may see Appendix note 1 below to get stats on `inserted_at` field for your form/case data.
+      - The start_date and end_date are optional and inclusive.
+      - If the dates are not specified entire data is reindexed in a single command.
+      - The command doesn't support pause/resume, so each time it reindexes all the data in the specified date range again, even if the data is already reindexed.
+   3. Below is a sequence of commands for an example scenario 
+      ```
+      # reindex all of form data up to Jan 2016.
+      ./manage.py reindex_es_from_remote forms http://es2_cluster:9200 
+      --end_date=2016-01-01
+      # reindex form data from Jan 2016 to October 2020
+      ./manage.py reindex_es_from_remote forms http://es2_cluster:9200 
+      --start_date=2016-01-01 --end_date=2020-10-01
+      # reindex form data added newly since October 1st 2020 (this can be done in the switchover phase explained next) 
+      ./manage.py reindex_es_from_remote forms http://es2_cluster:9200 
+      --start_date=2020-10-01
```

Comment:
```suggestion
      # reindex all of form data up to Jan 2016.
      ./manage.py reindex_es_from_remote forms http://es2_cluster:9200 --end_date=2016-01-01
     
       # reindex form data from Jan 2016 to October 2020
      ./manage.py reindex_es_from_remote forms http://es2_cluster:9200 --start_date=2016-01-01 --end_date=2020-10-01

      # reindex form data added newly since October 1st 2020 (this can be done in the switchover phase explained next) 
      ./manage.py reindex_es_from_remote forms http://es2_cluster:9200 --start_date=2020-10-01
```